torax/_src/config/config_loader.py

```diff
@@ -25,6 +25,10 @@
 from torax._src import path_utils
 from torax._src.plotting import plotruns_lib
 from torax._src.torax_pydantic import model_config
+from torax._src.config.runtime_validation_utils import (
+  _check_q_profile,
```

Comment:
Can we move these to be pydantic validators within the model_config file instead of here